docs/RunningOnDeviceIOS.md

```diff
@@ -28,19 +28,24 @@ You can also pack all the JavaScript code within the app itself. This way you ca
 1. Open `AwesomeApp/ios/AwesomeApp/AppDelegate.m`
 2. Follow the instructions for "OPTION 2":
   * Uncomment `jsCodeLocation = [[NSBundle mainBundle] ...`
-  * Run the `react-native bundle` command in terminal from the root directory of your app
+  * Run the `react-native bundle --platform <ios|android> --entry-file <path> --bundle-output <path> [--sourcemap-output path> --dev --minify]` command in terminal from the root directory of your app
```

Comment:
After https://github.com/facebook/react-native/commit/be70e32de2c417a467af902e7f4e2a0058c8e171 there is no need to do that on iOS, the bundle is generated as part of Xcode build.